src/lib/components/Link.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+export class Link extends React.PureComponent<React.HTMLProps<{}>> {
+  handleClick: React.MouseEventHandler<{}> = (e) => {
+    if (this.props.onClick) {
+      this.props.onClick(e)
+
+      if (e.defaultPrevented) {
+        return
+      }
+    }
+
+    if (!isModifiedClick(e)) {
+      return
+    }
+
+    // e.preventDefault()
+  }
+
+  render() {
+    return (
+      <a href={this.props.href} onClick={this.handleClick}>
```

Comment:
Will this cause full page refreshes when clicking a link? Or is it doing react-router like redirects?

Reply:
Yeah, it's a placeholder component. Will document.